patches/0009-Remove-duplicative-resource-token.patch

```diff
@@ -0,0 +1,29 @@
+From 0aed0d1880ef9501f931bae144ef8e0fed8e899a Mon Sep 17 00:00:00 2001
+From: Aaron Friel <mayreply@aaronfriel.com>
+Date: Thu, 22 Jun 2023 16:38:12 -0700
+Subject: [PATCH 9/9] Remove duplicative resource token
+
+---
+ google-beta/provider.go | 6 ++++++
+ 1 file changed, 6 insertions(+)
+
+diff --git a/google-beta/provider.go b/google-beta/provider.go
+index cf8f23f49..52b5e5ddf 100644
+--- a/google-beta/provider.go
++++ b/google-beta/provider.go
+@@ -792,6 +792,12 @@ func Provider() *schema.Provider {
+ 
+ func DatasourceMap() map[string]*schema.Resource {
+ 	datasourceMap, _ := DatasourceMapWithErrors()
++
++	datasourcesToRemove := []string{"google_organization_iam_policy"}
```

Comment:
The PR description mentions that `google_organizations_iam_policy_legacy` was also patched out but I don't see it here, is that somewhere else?

Reply:
Will add that.